src/fs.rs

```diff
@@ -154,8 +154,10 @@ fn consistent_access_fs_rw() {
         let access_all = AccessFs::from_all(abi);
         let access_read = AccessFs::from_read(abi);
         let access_write = AccessFs::from_write(abi);
+        let access_file = AccessFs::from_file(abi);
         assert_eq!(access_read, !access_write & access_all);
         assert_eq!(access_read | access_write, access_all);
+        assert_eq!(access_file, access_all & ACCESS_FILE);
```

Comment:
This is exactly what does `from_file()` but we can add that to tests.

Reply:
Course of action unclear. Followed a similar approach to `assert_eq!(access_read, !access_write & access_all);` ("regression test").